common/selectors/selectAndDedupe.js

```diff
@@ -13,14 +13,12 @@ const dedupe = require("lib/dedupe");
 module.exports = function selectAndDedupe(group) {
   return group.reduce((result, options, index, arr) => {
     let current;
+    let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;
```

Comment:
Cool. Let's make sure the doc is up-to-date please.

It took me some time to understand this function :). For example, initially, I was assuming the `dedupe.group()` function was implemented as "sort | uniq", which might include the defaults incorrectly in some cases. Fortunately, that's not the case 😄 !